Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml

```diff
@@ -0,0 +1,1497 @@
+      type: String
+    - contextPath: QRadar.Reference.Name
+      description: Name of the reference set.
+      type: String
+    - contextPath: QRadar.Reference.ElementType
+      description: Type of the elements in the reference set.
+      type: String
+    arguments:
+    - name: ref_name
+      required: false
+      description: The reference name of the reference set to retrieve its details.
+        Specify ref_name if you want to get details about certain reference set.
+    - name: range
+      description: 'Range of results to return (e.g.: 0-20, 3-5, 3-3).'
+      defaultValue: 0-49
+    - name: filter
+      description: 'Query to filter reference sets. For reference please consult:
+        https://www.ibm.com/support/knowledgecenter/SS42VS_SHR/com.ibm.qradarapi.doc/c_rest_api_filtering.html'
+    - name: fields
+      description: 'If used, will filter all fields except for the specified ones.
+        Use this parameter to specify which fields you would like to get back in the
+        response. Fields that are not explicitly named are excluded. Specify subfields
+        in brackets and multiple fields in the same object separated by commas. For
+        full list of available fields, https://www.ibm.com/support/knowledgecenter/SS42VS_SHR/com.ibm.qradarapi140.doc/14.0--reference_data-sets-GET.html'
+      isArray: true
+  - name: qradar-reference-set-create
+    description: Create a new reference set.
+    outputs:
+    - contextPath: QRadar.Reference.TimeoutType
+      description: Timeout type of the reference set. One of 'UNKNOWN', 'FIRST_SEEN',
+        'LAST_SEEN'.
+      type: String
+    - contextPath: QRadar.Reference.NumberOfElements
+      description: Number of elements in the reference set.
+      type: Number
+    - contextPath: QRadar.Reference.TimeToLive
+      description: Time left to live for reference.
+      type: String
+    - contextPath: QRadar.Reference.Data.LastSeen
+      description: Date when this data was last seen.
+      type: String
+    - contextPath: QRadar.Reference.Data.FirstSeen
+      description: Date when this data was first seen.
+      type: String
+    - contextPath: QRadar.Reference.Data.Source
+      description: Source of this data.
+      type: String
+    - contextPath: QRadar.Reference.Data.Value
+      description: Data value.
+      type: String
+    - contextPath: QRadar.Reference.CreationTime
+      description: Date when reference set was created.
+      type: String
+    - contextPath: QRadar.Reference.Name
+      description: Name of the reference set.
+      type: String
+    - contextPath: QRadar.Reference.ElementType
+      description: Type of the elements in the reference set.
+      type: String
+    arguments:
+    - name: ref_name
+      required: true
+      description: The name of the reference set to be created. Reference names can
+        be found by 'Name' field in 'qradar-reference-sets-list' command.
```

Comment:
i think you refer here to the command by mistake
if i understand correctly, we cant use a name that is already taken